state-chain/pallets/cf-elections/src/lib.rs

```diff
@@ -149,7 +151,18 @@ pub mod pallet {
 	#[derive(
 		PartialEq, Eq, PartialOrd, Ord, Clone, Copy, Debug, Encode, Decode, TypeInfo, Default,
 	)]
-	struct UniqueMonotonicIdentifier(u64);
+	pub struct UniqueMonotonicIdentifier(u64);
+
+	#[cfg(test)]
+	impl UniqueMonotonicIdentifier {
+		pub(crate) fn new(value: u64) -> Self {
```

Comment:
This combination pub and pub(crate) means anyone can access the type, but can't construct it. Is this intended? How do we expect the engine to construct a vote for an election?

Reply:
The engine will get the identifier from the subscription that will be based on the `validator_election_data` function, it should never need to construct it